diff --git a/src/app/models/app-state.ts b/src/app/models/app-state.ts
index d5460fa..a561225 100644
--- a/src/app/models/app-state.ts
+++ b/src/app/models/app-state.ts
@@ -15,5 +15,5 @@ export class AppState extends Model {
   @autoserialize swipeCounting = true;
   @autoserialize recordPosition = true;
   @autoserialize vibrate = true;
-  @autoserialize directMonitization = true;
+  @autoserialize directMonitization = false;
 }
